tests/components/hassio/test_binary_sensor.py

```diff
@@ -205,8 +205,10 @@ async def test_binary_sensor(
     expected,
     aioclient_mock: AiohttpClientMocker,
     entity_registry: er.EntityRegistry,
+    addon_installed: AsyncMock,
 ) -> None:
     """Test hassio OS and addons binary sensor."""
+    addon_installed.return_value.state = "started" if expected == "on" else "stopped"
```

Comment:
We don't want if statements in the test, so rather add another parameter for the returned state